app/scripts/views/mixins/flow-begin-mixin.js

```diff
@@ -2,27 +2,17 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-// Tells the server to emit the `flow.begin` flow event.
+// Tells the server to emit the `flow.${viewName}.begin` event.
+// THIS MUST BE MIXED IN *AFTER* views/mixins/flow-events-mixin.
```

Comment:
Exactly, yep. We want to use `flow-events-mixin` independently but my attempts to re-use it in this mixin didn't work out. We can't use cocktail because it invokes the colliding `afterRender` implementations in the wrong order and I also tried composition but it was fugly.